ember-cli-rx/addon/schedulers/ember-scheduler.js

```diff
@@ -28,10 +28,12 @@ function scheduleAbsolute(state, dueTime, action) {
 }
 
 function scheduleEmberAction(disposable, queue, target, state, action, scheduler) {
-	Ember.run.schedule(queue, target, function() {
-		if(!disposable.isDisposed) {
-			disposable.setDisposable(action(scheduler, state));
-		}
+	Ember.run(function() {
```

Comment:
i believe: https://github.com/blesh/RxEmber/blob/b5f66a4ce1249d385a4b2d0f7d5b0e3adb6f6a88/ember-cli-rx/addon/schedulers/ember-scheduler.js#L20 needs the run-loop